ui/apps/platform/src/Containers/Dashboard/PatternFly/Widgets/ImagesAtMostRiskTable.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import { HashLink as Link } from 'react-router-hash-link';
+import { TableComposable, Thead, Tr, Th, Tbody, Td } from '@patternfly/react-table';
+import { SecurityIcon } from '@patternfly/react-icons';
+
+import { ImageName } from 'types/image.proto';
+import { severityColors } from 'constants/visuals/colors';
+import { middleTruncate } from 'utils/textUtils';
+import { vulnManagementPath } from 'routePaths';
+import { Tooltip } from '@patternfly/react-core';
+
+type VulnCounts = {
+    total: number;
+    fixable: number;
+};
+
+export type ImageData = {
+    images: {
+        id: string;
+        name: Partial<ImageName>;
+        priority: number;
+        vulnCounter: {
+            important: VulnCounts;
+            critical: VulnCounts;
+        };
+    }[];
+};
+
+export type CveStatusOption = 'Fixable' | 'All';
+
+export type ImagesAtMostRiskProps = {
+    imageData: ImageData;
+    cveStatusOption: CveStatusOption;
+};
+
+const columnNames = {
+    imageName: 'Images',
+    riskPriority: 'Risk priority',
+    criticalCves: 'Critical CVEs',
+    importantCves: 'Important CVEs',
+};
+
+function linkToImage(id: string) {
+    return `${vulnManagementPath}/image/${id}#image-findings`;
+}
+
+function ImagesAtMostRiskTable({ imageData: { images }, cveStatusOption }: ImagesAtMostRiskProps) {
+    return (
+        <TableComposable aria-label="Images at most risk" variant="compact" borders={false}>
+            <Thead>
+                <Tr>
+                    <Th width={35} className="pf-u-pl-0">
+                        {columnNames.imageName}
+                    </Th>
+                    <Th className="pf-u-text-align-center-on-md">{columnNames.riskPriority}</Th>
+                    <Th>{columnNames.criticalCves}</Th>
+                    <Th className="pf-u-pr-0">{columnNames.importantCves}</Th>
+                </Tr>
+            </Thead>
+            <Tbody>
+                {images.map(({ id, name, priority, vulnCounter }) => (
+                    <Tr key={id}>
+                        <Td className="pf-u-pl-0" dataLabel={columnNames.imageName}>
+                            <Link
+                                to={linkToImage(id)}
+                                scroll={(el: HTMLElement) =>
+                                    // TODO This is a heavy handed way to scroll to the CVE section which is loaded on
```

Comment:
One of the requirements is that the link out from here scrolls to the "Image Findings" section in the VulnMgmt image page, i.e. the PatternFly style CVE tables. The problem is that the automatic scroll position is lost once data on that page loads data asynchronously.

The timeout here is a bit of a hack to preserve the scroll position somewhat reliably without having to build more sophisticated "hash scrolling" into the soon to be replaced Vuln Mgmt pages.

The `500` ms threshold is a trade off. Lower than that and we risk missing the net request and not scrolling correctly, longer and the page scrolling into place is jarring for the user.

Definitely open to suggestions on a cleaner way to do this if anyone has any ideas.